hopsworks-common/src/main/java/io/hops/hopsworks/common/jupyter/JupyterController.java

```diff
@@ -72,4 +120,126 @@ public void convertIPythonNotebook(String hdfsUsername, String path, Project pro
           ex);
     }
   }
+
+  @TransactionAttribute(TransactionAttributeType.REQUIRES_NEW)
```

Comment:
@robzor92 Why is this method in a transaction? it goes to the filesystem and we might get timeouts here, no?

Reply:
Accidently put it there, will remove it. Did not have issues with timeout but I suppose it may happen.

Reply:
Removed it